openlibrary/plugins/upstream/account.py

```diff
@@ -402,6 +403,43 @@ def GET(self):
         # Show failed message without thinking.
         return render['account/verify/failed']()
 
+class account_validation(delegate.page):
+    path = '/account/validate'
+
+    @staticmethod
+    def validate_username(username):
+        if not username.isalnum():
+            return 'Username may only contain numbers and letters'
+        if not (21 > len(username) > 2):
+            return 'Username must be between 3-20 characters'
```

Comment:
We probably want to use the same regex as in other username checks (I believe we used to allow `-` and `_`)

Reply:
Confirmed: https://github.com/internetarchive/openlibrary/pull/3472/files#diff-64069c0678ce3631f1e8346c6630cceaR31

Reply:
Okay then will update it